tests/navier_stokes-chorin-temam_00.cc

```diff
@@ -0,0 +1,43 @@
+#include "pidomus.h"
+#include "interfaces/navier_stokes-chorin-temam.h"
+#include "tests.h"
+
+/**
+ * Test:     Navier Stokes interface.
+ * Method:   Direct
+ * Problem:  Non time depending Navier Stokes Equations
+ * Exact solution:
+ * \f[
+ *    u=\big( 2*(x^2)*y, -2*x*(y^2) \big)
+ *    \textrm{ and }p=xy;
+ * \f]
+ */
+
+using namespace dealii;
+int main (int argc, char *argv[])
+{
+
+  Utilities::MPI::MPI_InitFinalize mpi_initialization(argc, argv,
+                                                      numbers::invalid_unsigned_int);
+
+  initlog();
+  deallog.depth_file(1);
+
+  const int dim = 2;
+
+  NavierStokes<dim, dim, LADealII> energy;
```

Comment:
Se faccio sprintare qualcosa dove usa UMFPACK sembra andare.
Però quando non converge mi da l’errore di Kinsol

> On 19 Jul 2016, at 14:47, Alberto Sartori <notifications@github.com> wrote:
> 
> In tests/navier_stokes-chorin-temam_00.cc <https://github.com/mathLab/pi-DoMUS/pull/214#discussion_r71330310>:
> 
> > + * \f]
> > + */
> > +
> > +using namespace dealii;
> > +int main (int argc, char *argv[])
> > +{
> > +
> > +  Utilities::MPI::MPI_InitFinalize mpi_initialization(argc, argv,
> > +                                                      numbers::invalid_unsigned_int);
> > +
> > +  initlog();
> > +  deallog.depth_file(1);
> > +
> > +  const int dim = 2;
> > +
> > +  NavierStokes<dim, dim, LADealII> energy;
> invece in parallelo cosa chiama? Vedendo il file di parametri, di default viene usato imex con kinsol
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/mathLab/pi-DoMUS/pull/214/files/94b5e0aaa523f18df661ca4d04322d48edfafaee#r71330310>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AIisRZn9MuwWkXx0t0ljxeL9m3KweUkuks5qXMdVgaJpZM4JLR9n>.
>